docs/integrations/ember-and-json-api.md

```diff
@@ -17,8 +17,30 @@ To solve this, in Ember, both the adapter and the serializer will need some modi
 
 ### Server-Side Changes
 
-there are multiple mimetypes for json that should all be parsed similarly, so
+First, set the adapter type in an initializer file:
+
+```ruby
+# config/initializers/ams_config.rb
+ActiveModelSerializers.config.adapter = :json_api
+```
+
+or:
+
+```ruby
+# config/initializers/ams_config.rb
+ActiveModelSerializers.config.adapter = ActiveModelSerializers::Adapter::JsonApi
+```
+
+You will also want to set the `key_transform` to `:underscore` since you will adjust the attributes in your Ember serializer to use underscores instead of dashes later.
```

Comment:
I'd say the bigger point here is that if you'd like to handle key transform on the client (Ember) side, to use `key_transform: :unaltered` since it's much faster.